modules/installation-and-upgrade/pages/container-deployment/suma/server-migration-suma.adoc

```diff
@@ -59,33 +64,62 @@ systemctl stop postgresql
 == Prepare the SSH Connection
 
 .Procedure: Preparing the SSH connection
-. The SSH configuration and agent should be ready on the new {productnumber} server for a passwordless connection to the 4.3 server.
+
+. Ensure that for [systemitem]``root`` an SSH key exists on the new {productnumber} server.
+  If a key does not exist, create it with:
++
+
+----
+ssh-keygen -t rsa
+----
+
+. The SSH configuration and agent should be ready on the new server for a passwordless connection to the 4.3 server.
 +
 
 [NOTE]
 ====
 To establish a passwordless connection, the migration script relies on an SSH agent running on the {productnumber} server.
 If the agent is not active yet, initiate it by running [command]``eval $(ssh-agent)``.
-Then, add the SSH key to the running agent with [command]``ssh-add /path/to/the/private/key``.
+Then, add the SSH key to the running agent with [command]``ssh-add``; if the key is not in a standard file, provide it as an argument such as [command]``ssh-add </path/to/the/private/key>``.
```

Comment:
"if the key is not in a standard file" ???

this sounds plain wrong to me -- where does that come from ?

suggested:

Then, add the SSH key to the running agent with [command]``ssh-add`` followed by the path to the private key.

Reply:
ok